### ISSUE TITLE
make yii-mongo composer dependency as suggestion,not requirement

### ISSUE THREAD

**Issue by Insolita** [joni-jones/yii2-wschat#21]
if chat support other database storage, why strict dependency of yii2-mongo (yii2-mongo requires ext-mongo - so force to reconfigure environment)

**Comment by coolmut** [comment]
Согласен, yii2-mongo не ставится без драйвера mongo, который некоторым совсем не нужен. Отсюда возникает проблема, при которой необходимо сначала поставить этот драйвер, после чего зависимость сможет подтянуться.

**Comment by Insolita** [comment]
@coolmut Судя по тому что уже прошел почти год с вопроса, автор забил.
